aries-site/src/layouts/main/Layout.js

```diff
@@ -53,15 +61,79 @@ export const Layout = ({
   const layout = isLanding ? 'plain' : pageLayout;
 
   const MainContentWrapper = isLanding ? Fragment : PageContent;
-  const size = useContext(ResponsiveContext);
+  const breakpoint = useContext(ResponsiveContext);
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  const [value, setValue] = useState();
+  const [surveyResults, setSurveyResults] = useState({});
+
+  const ratingValue = surveyResults[0];
+  const textValue = surveyResults[1];
+  const feedbackValues = {
+    ratingValue: parseInt(value?.['like-rating']),
+    textValue: value?.['text-area'],
+  };
+
+  useEffect(() => {
+    fetch(
+      `${REACT_FEEDBACK_SURVEY}SV_ezVvKtA2bBBbvDM/responses/R_1HitDZLeYlmyx3E`,
+      {
+        method: 'GET',
+        headers: {
+          'X-API-TOKEN': 'acLCL6dZED3y8nZhqPSq1R6rCTFh1adqrYEXPdfY',
```

Comment:
This should probably be in a `.env`, especially since it's used more than once. And then later we could make it a github secret if needed.

Reply:
done